src/libraries/System.Text.Json/src/System/Text/Json/Serialization/JsonConverterOfT.cs

```diff
@@ -251,6 +256,7 @@ internal bool TryRead(ref Utf8JsonReader reader, Type typeToConvert, JsonSeriali
                 Debug.Assert(this is ObjectConverter);
                 success = OnTryRead(ref reader, typeToConvert, options, ref state, out value);
                 Debug.Assert(success);
+                populatedValue = default;
```

Comment:
Same here
```suggestion
                isPopulatedValue = false;
```
This makes me think we're setting a populatedValue of type `T` to its default value.

Reply:
will be included in the next iteration